tests/test_layered_image.py

```diff
@@ -224,7 +224,7 @@ def test_grow_mask_mult(self):
         mask.set_pixel(10, 12, 1)
         self.image.set_mask(mask)
         self.image.apply_mask_flags(1, [])
-        self.image.grow_mask(3, True)
+        self.image.grow_mask(3)
```

Comment:
This case strikes me as odd, and I'm not sure why it's not combined with the one above it. Maybe it's just that the below line ("Check that the mask has grown to all applicable pixels") is not a good description of what's being verified with this case.

Outside the scope of this change, though.

Reply:
That's a good point. There is the potential to combine these two in the future.